### PR TITLE
docs(security): remove db reference in bruteforce + add logs

### PR DESCRIPTION
**Pull request by joshtrichards**
### ☑️ Resolves

Troubleshooting updates:

- Clarify log level instructions
- Add example log entries to look for
- Drop db table reference since it's misleading given we now use the memcache whenever possible

### 🖼️ Screenshots

image

### CODE REVIEW

File: admin_manual/configuration_server/bruteforce_configuration.rst [hunk 1]
```diff
@@ -121,22 +121,31 @@ Troubleshooting
 Overview
 ~~~~~~~~
 
-On most setups Nextcloud will work out of the box without any issues. If you
-run into a situation where logging in or connecting is often very slow for multiple users, the first
-step is to check your Nextcloud Server logs to see what IP addresses are being detected (you may need 
-adjust your logging to INFO level temporarily to do so). 
+On most setups Nextcloud will work out of the box without any issues. If you run into a situation where 
+logging in or connecting is often very slow for multiple users, the first step is to check your Nextcloud 
+Server logs to see what IP addresses are being detected (you will need  adjust your ``loglevel`` to ``1``
```
Comment (nickvergessen) [inline admin_manual/configuration_server/bruteforce_configuration.rst:126]:
```suggestion
Server logs to see what IP addresses are being detected (you will need adjust your ``loglevel`` to ``1`` 
```

File: admin_manual/configuration_server/bruteforce_configuration.rst [hunk 2]
```diff
@@ -121,22 +121,31 @@ Troubleshooting
...
+- `IP address throttled` [...]
+- `IP address blocked` [...]
 
 If all clients appear to be coming from the same IP address and that IP address happens to be your 
-proxy, you need to review your ``trusted_proxies`` configuration. If it is a common connection point,
-such as a multi-user office location, it is possible whitelisting is appropriate.
+proxy, you need to review your ``trusted_proxies`` configuration. 
+
+If the IP address is a common connection point, such as a multi-user office location, it is possible whitelisting 
+is appropriate.
```
Comment (nickvergessen) [inline admin_manual/configuration_server/bruteforce_configuration.rst:139]:
```suggestion
If the IP address is a common connection point, such as a multi-user office location, it is possible allow-listing it.
```

Comment (nickvergessen) [reply admin_manual/configuration_server/bruteforce_configuration.rst:139]: at last the later half needs adjusting as it's not a correct sentence :S (mostlikely was me)

**Comment by kesselb** [comment]
Thank you :+1: 

A good idea to transfer the knowledge from the forum post to our documentation.

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**